### DISCUSSION TITLE
help + suggest

### DISCUSSION THREAD

**Discussion by triplefe** [geocine/sd-easy-mode#10] [General]
help =
what code needs to be changed so that the result is not large? I did it by default and the result is almost 2gb. :(

suggest =
if you can make a result for the safetensor option. there is only ckpt. it would be great if you can choose to save it as ckpt / safetensor.
or
or enter the converter ckpt to safetensor option and vice versa.
(first choice is pretty good...)

**Comment by geocine** [comment]
Smallest result will always be 2GB no matter what code you use. That means if you do not use this repository all other code that you will see or use online , this will be always the lowest size. If you want smaller size , you can explore other technic called LoRA (This repository is not LoRA this is Dreambooth). You can convert the ckpt to safetensor using the stable diffusion web ui plugin https://github.com/Akegarasu/sd-webui-model-converter
